src/weatherreport/README.md

```diff
@@ -0,0 +1,86 @@
+# Weather Report
```

Comment:
Most of this should find its way into a `couchdb-documentation` PR as well. 😉

Reply:
I looked at the `couchdb-documentation` repo, but didn't see a 3.x branch. It occurs to me that with the inclusion of weatherreport in 3.x (but not main) we probably should create one now?

Reply:
We haven't started merging 4.x documentation yet, so just use `main` for now.

As soon as we fork `3.x` for docs it's going to be much more annoying to maintain, esp. as typo fixes and so on keep coming in.